cores/esp8266/core_esp8266_waveform_pwm.cpp

```diff
@@ -264,54 +262,56 @@ IRAM_ATTR bool _stopPWM(uint8_t pin) {
   return _stopPWM_bound(pin);
 }
 
-static void _addPWMtoList(PWMState &p, int pin, uint32_t val, uint32_t range) {
-  // Stash the val and range so we can re-evaluate the fraction
-  // should the user change PWM frequency.  This allows us to
-  // give as great a precision as possible.  We know by construction
-  // that the waveform for this pin will be inactive so we can borrow
-  // memory from that structure.
+static void _addPWMtoList(PWMState& p, uint8_t pin, uint32_t val, uint32_t range) {
+    // Stash the val and range so we can re-evaluate the fraction
+    // should the user change PWM frequency.  This allows us to
+    // give as great a precision as possible.  We know by construction
+    // that the waveform for this pin will be inactive so we can borrow
+    // memory from that structure.
   wvfState.waveform[pin].desiredHighCycles = val;  // Numerator == high
   wvfState.waveform[pin].desiredLowCycles = range; // Denominator == low
 
-  uint32_t cc = (_pwmPeriod * val) / range;
+    uint32_t cc = (_pwmPeriod * val) / range;
```

Comment:
Everything but line 271/272 was reindented here.  Can you make them all the same indent?

Reply:
Blech, completely different presentation in my environment, lines 273-274 are mis-indented here - good catch of course, I'm fixing this, thanks.